Multisig/Data/Services/Safe Client Gateway Service/SCGModels.swift

```diff
@@ -14,6 +14,68 @@ struct Page<T: Decodable>: Decodable {
     let results: [T]
 }
 
+protocol TransactionSummaryItem: Decodable { }
+
+
+struct TransactionSummaryItemDateLabel: TransactionSummaryItem {
+    let timestamp: Int
+}
+
+struct TransactionSummaryItemLabel: TransactionSummaryItem {
+    let label: String
+}
+
+
+struct TransactionSummaryItemTransaction: TransactionSummaryItem {
+    let transaction: TransactionSummary
+    let conflictType: ConflictType
+}
+
+struct TransactionSummaryItemConflictHeader: TransactionSummaryItem {
+    let nonce: UInt256String
+}
+
+enum TransactionSummaryItemWrapper: Decodable {
```

Comment:
There's a new way I found how to use this, without a protocol but using just enum directly - please see the SCG.swift or let's have a call and I can assist you